docs/outlook/online-meeting.md

```diff
@@ -0,0 +1,182 @@
+        </MobileFormFactor>
+      </Host>
+    </Hosts>
+    ...
+  </VersionOverrides>
+</VersionOverrides>
+...
+```
+
+### Implementation
+
+In this section, you'll see an outline for how your add-in script can update a user's meeting to include online meeting details.
+
+The following example shows how you could construct online meeting details. Not shown is how to get the meeting organizer's ID and other details from your service.
+
+```js
+const newBody = '<br>' +
+    '<a href="https://contoso.com/meeting?id=123456789" target="_blank">Join Contoso meeting</a>' +
+    '<br><br>' +
+    'Phone Dial-in: +1(123)456-7890' +
+    '<br><br>' +
+    'Meeting ID: 123 456 789' +
+    '<br><br>' +
+    'Want to test your video connection?' +
+    '<br><br>' +
+    '<a href="https://contoso.com/testmeeting" target="_blank">Join test meeting</a>' +
+    '<br><br>';
+```
+
+The next example shows how you could define a UI-less function named `insertContosoMeeting` referenced in the manifest to update the meeting body with the online meeting details.
+
+```js
+var mailboxItem;
+
+// Office is ready.
+Office.onReady(function () {
+        mailboxItem = Office.context.mailbox.item;
+    }
+);
+
+function insertContosoMeeting(event) {
+    // Get HTML body from the client.
+    mailboxItem.body.getAsync("html",
+        { asyncContext: event },
+        function (getBodyResult) {
+            if (getBodyResult.status === Office.AsyncResultStatus.Succeeded) {
+                updateBody(getBodyResult.asyncContext, getBodyResult.value);
+            } else {
+                console.error("Failed to get HTML body.");
+                getBodyResult.asyncContext.completed({ allowEvent: false });
+            }
+        }
+    );
+}
+```
+
+This sample shows an implementation of the supporting function `updateBody` used in the previous example that appends the online meeting details to the current body of the meeting.
+
+```js
+function updateBody(event, existingBody) {
+    // Append new body to the existing body.
+    mailboxItem.body.setAsync(existingBody + newBody,
+        { asyncContext: event, coercionType: "html" },
+        function (setBodyResult) {
+            if (setBodyResult.status === Office.AsyncResultStatus.Succeeded) {
+                setBodyResult.asyncContext.completed({ allowEvent: true });
+            } else {
+                console.error("Failed to set HTML body.");
+                setBodyResult.asyncContext.completed({ allowEvent: false });
+            }
+        }
+    );
+}
+```
+
+### Testing and validation
+
+You can follow the usual guidance to [test and validate your add-in](testing-and-tips.md). After [sideloading](sideload-outlook-add-ins-for-testing.md) in Outlook on the web, Windows, or Mac, you should restart Outlook on your mobile device (Android is the only supported client for now) then verify on a new meeting screen that the Microsoft Teams or Skype toggle has been replaced with your own.
+
+#### Create meeting UI
```

Comment:
SEO and clarity: Consider removing H2 ## "Scenario: online meeting provider" and moving everything up a heading level so there's no heading 4